crates/consensus/auto-seal/src/lib.rs

```diff
@@ -115,13 +115,23 @@ where
         pool: Pool,
         to_engine: UnboundedSender<BeaconEngineMessage>,
         canon_state_notification: CanonStateNotificationSender,
+        block_max_transactions: Option<usize>,
+        block_time: Option<Duration>,
```

Comment:
these should be mutually exclusive, but I can see how this would be more convenient coming from CLI

so at the very least, this needs to be included in the docs

Reply:
I was not sure how to handle this. Throw if both are Some? Merge them to one enum parameter including both variants?